core/src/main/java/org/geysermc/geyser/inventory/recipe/TrimRecipe.java

```diff
@@ -82,7 +82,11 @@ public static TrimPattern readTrimPattern(GeyserSession session, RegistryEntry e
     private static String stripNamespace(String identifier) {
         int i = identifier.indexOf(':');
         if (i >= 0) {
-            return identifier.substring(i + 1);
+            String namespace = identifier.substring(0, i);
+            // Only strip minecraft namespace
+            if (namespace.equals("minecraft")) {
```

Comment:
considering this method now only removes the Minecraft namespace, it's probably a good idea to rename it to `stripMinecraftNamespace` just in case

Reply:
Sounds good, will apply in a moment